scrapers/rb_common/logger.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Stash
+  class Logger
+    # Log messages sent from a script scraper instance are transmitted via stderr
+    # and are encoded with a prefix consisting of the special character SOH, then
+    # the log level (one of t, d, i, w, or e - corresponding to trace, debug, info,
+    # warning and error levels respectively), then the special character STX.
+
+    def trace(text)
+      log("t", text)
+    end
+
+    def debug(text)
+      log("d", text)
+    end
+
+    def info(text)
+      log("i", text)
+    end
+
+    def warning(text)
+      log("w", text)
+    end
+
+    def error(text)
+      log("e", text)
+    end
+
+    private
+
+    def log(level, text)
+      level_char = control_wrap(level)
+
+      # I'm not sure what case is covered by the image part of this regex, but it
+      # was present in the py_common version so I've included it.
```

Comment:
There was an issue when trying to print to the console through the log function a base64 encoded image. It could be python's issue or a console print limitation. Eitherway it caused the scrapers to error out so every attemt to log print a base64 scraped image was replaced with `[...]`

Reply:
Hmmm I'll test that out, cause if it isn't needed, I'd love to remove it, and if it is needed, I'll add a proper comment here with why it is needed.

Reply:
Do you know of a test case where this would happen? Having trouble finding a way to replicate